cli/src/config.ts

```diff
@@ -22,6 +22,9 @@ export interface PlayerConfigResolvedShape {
 
     /** Flag to omit validating the resulting JSON */
     skipValidation?: boolean;
+
+  /** Array of file names to skip during compilation */
+   skipFiles?: string[];
```

Comment:
I think allowing the the list of files to be provided here could cause some confusion because this config should apply to all CLI commands.